modules/spring-data/src/main/java/org/apache/ignite/springdata/repository/support/ConditionFalse.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.ignite.springdata.repository.support;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+/**
+ * Always false condition.
+ * Tells spring context never load bean with such Condition.
+ *
+ * @author Roman_Meerson
```

Comment:
Remove `@author`, because Apache coding standards recommends to not left names in the class header for a good reasons.
See http://directory.apache.org/fortress/coding-standards.html#classinterface-headers